template/web/wp-content/themes/<%= slug %>/templates/components/pagination.twig

```diff
@@ -1,60 +1,61 @@
 {% if posts.pagination.pages is not empty %}
-    <nav class="pagination-block">
-        <ul class="pagination">
-            {% if pagination.pages|first and pagination.pages|first.current != true %}
-                <li class="first btn">
-                    <a href="{{ pagination.pages|first.link }}">First</a>
-                </li>
-            {% else %}
-                <li class="first btn disabled">
-                    <button disabled>First</button>
-                </li>
-            {% endif %}
+  <nav class="pagination-block">
+    <ul class="pagination">
+      {% if pagination.pages|first and pagination.pages|first.current != true %}
```

Comment:
Need to double check this file, it's containing some errors

Reply:
@perruche Lets talk about this on monday!

Reply:
What are the errors? Any idea when it could be fixed? I would like to merge every PR to release a new version before our next WP projects.

Reply:
There is some missing `posts` values in the file:
like : `{% if POSTS.pagination.pages|first and POSTS.pagination.pages|first.current != true %}`